src/wasm.h

```diff
@@ -1128,6 +1129,16 @@ class RefFunc : public SpecificExpression<Expression::RefFuncId> {
   void finalize();
 };
 
+class RefEq : public SpecificExpression<Expression::RefEqId> {
```

Comment:
I was thinking about whether or not it would be better to make this a new `BinaryOp` rather than a new expression class, but it seems to me that for now its uses will probably be more different from than similar to uses of most other `BinaryOp`s, so I think making it a new expression class makes sense.